src/api/app/models/package.rb

```diff
@@ -1457,6 +1460,28 @@ def last_build_reason(repo, arch, package_name = nil)
     PackageBuildReason.new(data)
   end
 
+  # Methods for Thinking Sphinx real-time indexes
```

Comment:
Can you isolate them in some file please instead of working around the ClassLength cop.

Reply:
Done. Now they are isolated in `app/mixins/package_sphinx.rb`.

Reply:
However adding the two `include` calls made `app/models/package.rb` bigger, so I changed the `Max` limit of the `Metrics/ClassLength` cop.